src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+	protected String description;
+	protected boolean isDone;
+
+	public Task(String description) {
+		this.description = description;
+		this.isDone = false;
+	}
+
+	public void setDone() {
+		isDone = true;
+	}
+
+	public void setNotDone() {
```

Comment:
"Not" in a method can be misleading and not very readable -  consider replacing the `setDone` and `setNotDone` with this one method or something similar.
```
public void setIsDone(boolean value) {
		isDone = value;
	}
```
(you can also change the setter name for setIsDone in this case)